src/GoogleReader.m

```diff
@@ -207,62 +192,63 @@ -(void)addClientTokenToRequest:(ASIHTTPRequest *)clientRequest
+                    [self.clientAuthWaitQueue removeObject:obj];
                 }
+                [APPCONTROLLER setStatusMessage:nil persist:NO];
             } else {
                 NSString *response = [strongRequest responseString];
                 NSArray *components = [response componentsSeparatedByString:@"\n"];
 
                 //NSString * sid = [[components objectAtIndex:0] substringFromIndex:4];		//unused
                 //NSString * lsid = [[components objectAtIndex:1] substringFromIndex:5];	//unused
                 self.clientAuthToken = [NSString stringWithString:[components[2] substringFromIndex:5]];
+                googleReaderStatus = missingTToken;
 
-                for (id obj in clientAuthWaitQueue) {
+                for (id obj in self.clientAuthWaitQueue) {
                     [(ASIHTTPRequest *)obj addRequestHeader:@"Authorization" value:[NSString stringWithFormat:@"GoogleLogin auth=%@",
                                                                                  self.clientAuthToken]];
                 }
-                for (id obj in [clientAuthWaitQueue reverseObjectEnumerator]) {
-                    [clientAuthWaitQueue removeObject:obj];
+                for (id obj in [self.clientAuthWaitQueue reverseObjectEnumerator]) {
+                    [self.clientAuthWaitQueue removeObject:obj];
                 }
 
-                googleReaderStatus = missingTToken;
-                [[NSNotificationCenter defaultCenter] postNotificationOnMainThreadWithName:@"GRSync_Autheticated" object:nil];
-
-                if (clientAuthTimer == nil || !clientAuthTimer.valid) {
+                if (self.clientAuthTimer == nil || !self.clientAuthTimer.valid) {
                     //new request every 6 days
-                    clientAuthTimer = [NSTimer scheduledTimerWithTimeInterval:6 * 24 * 3600
-                                                                 target:self
-                                                               selector:@selector(resetAuthentication)
-                                                               userInfo:nil
-                                                                repeats:YES];
+                    self.clientAuthTimer = [NSTimer scheduledTimerWithTimeInterval:6 * 24 * 3600
+                                                                            target:self
+                                                                          selector:@selector(resetAuthentication)
+                                                                          userInfo:nil
+                                                                           repeats:YES];
                 }
+                // pause for a second
+                // to make sure dependent requests are launched only when the OpenReader server is ready
+                [[NSRunLoop currentRunLoop] runUntilDate:[NSDate dateWithTimeIntervalSinceNow:1]];
```

Comment:
This should impact performance much less than  `[NSThread sleepForTimeInterval:1]` would and is suggested at : 
- http://stackoverflow.com/questions/3260307/need-objective-c-pause-wait-function
- or http://stackoverflow.com/questions/1224784/how-do-i-pause-a-function-in-c-or-objective-c-on-mac-without-using-sleep

I am yet unsure if/why this approach is better than sending a notification as the original code does… I will suspend this pull request till I finish some verifications.